content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -8,15 +8,12 @@ weight: 60
 
 {{% capture overview %}}
 
-You can use a `PodPreset` object to inject information like secrets, volume
-mounts, and environment variables etc into pods at creation time.
-This task shows some examples on using the `PodPreset` resource.
+This page shows how to use Pod Presets to inject information like secrets, volume mounts, and environment variables into Pods at creation time.
```

Comment:
Hello @tom1299 .
I am wondering about the use of Pod Preset versus PodPreset throughout the page.

Reply:
Actually that was one of the points I was not sure of. In the [concepts page](https://kubernetes.io/docs/concepts/workloads/pods/podpreset/) "PodPreset" and "Pod Preset" are used interchangeably. And I am struggling to see the rule when to use which.

Reply:
I updated the page so that it now contains only "Pod Prese" (with a space). I also changed it in the title so that reflects the corresponding "concepts" page. Could you please take a look at it ?